From 5f8bf882c71439ddb7acb9908d59a5c6d93b696e Mon Sep 17 00:00:00 2001
From: Sailaja <sbandarupalli@habana.ai>
Date: Wed, 25 Oct 2023 15:07:09 +0300
Subject: [PATCH] [SW-0] gaudi3: pcie init: correct  bar4 offset

correct the typo in assginment of PCIE DBI BAR4 reg

Change-Id: Ia1a7ed62e1517bc3612716a331d9e299c8a2d310
Signed-off-by: Sailaja <sbandarupalli@habana.ai>
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index a0b2c2e..c395f0a 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -540,7 +540,7 @@
 		/* DIE 0 */
 		pci.dbi_bar0_reg = D0_PCIE_DBI_SIG_REG(mmPCIE_DBI_BAR0_REG);
 		pci.dbi_bar2_reg = D0_PCIE_DBI_SIG_REG(mmPCIE_DBI_BAR2_REG);
-		pci.dbi_bar4_reg = D0_PCIE_DBI_SIG_REG(mmPCIE_DBI_BAR2_REG);
+		pci.dbi_bar4_reg = D0_PCIE_DBI_SIG_REG(mmPCIE_DBI_BAR4_REG);
 		pci.dbi_status_command_reg = D0_PCIE_DBI_SIG_REG(mmPCIE_DBI_STATUS_COMMAND_REG);
 		pci.dbi_uncorr_aer_mask = D0_PCIE_DBI_SIG_REG(mmPCIE_DBI_UNCORR_ERR_MASK_OFF);
 		pci.aux_dbi_reg = D0_PCIE_AUX_REG(mmPCIE_AUX_DBI);
@@ -557,7 +557,7 @@
 		/* DIE 1 */
 		pci.dbi_bar0_reg = D1_PCIE_DBI_SIG_REG(mmPCIE_DBI_BAR0_REG);
 		pci.dbi_bar2_reg = D1_PCIE_DBI_SIG_REG(mmPCIE_DBI_BAR2_REG);
-		pci.dbi_bar4_reg = D1_PCIE_DBI_SIG_REG(mmPCIE_DBI_BAR2_REG);
+		pci.dbi_bar4_reg = D1_PCIE_DBI_SIG_REG(mmPCIE_DBI_BAR4_REG);
 		pci.dbi_status_command_reg = D1_PCIE_DBI_SIG_REG(mmPCIE_DBI_STATUS_COMMAND_REG);
 		pci.dbi_uncorr_aer_mask = D1_PCIE_DBI_SIG_REG(mmPCIE_DBI_UNCORR_ERR_MASK_OFF);
 		pci.aux_dbi_reg = D1_PCIE_AUX_REG(mmPCIE_AUX_DBI);
